src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+let total = 0;
```

Comment:
Since you're using `forEach` to calculate the total population, you could use the `reduce` method instead to make the code more functional and concise. 😊